lisa/analysis/tasks.py

```diff
@@ -365,6 +365,32 @@ def stringify_df_task_states(cls, df, columns, inplace=False):
 
         return df
 
+    @df_tasks_states.used_events
+    def df_tasks_runtime(self):
+        """
+        DataFrame of the time each task spent in TASK_ACTIVE (:class:`TaskState`)
+
+        :returns: a :class:`pandas.DataFrame` with:
+
+          * PIDs as index
+          * A ``runtime`` column (the time that task spent running)
+        """
+        df = self.df_tasks_states()
```

Comment:
I think for this `df` it could be very useful to add a column with the task's `comm`... I guess we will end up most of the time wondering _"what task is that PID"_ ?

Reply:
We can always get the PID from `get_task_pid()` but this dataframe should always be relatively small, so we can add that info.